resources/views/map.blade.php

```diff
@@ -7,20 +7,42 @@
 
         <script src='https://api.mapbox.com/mapbox-gl-js/v0.44.2/mapbox-gl.js'></script>
         <link href='https://api.mapbox.com/mapbox-gl-js/v0.44.2/mapbox-gl.css' rel='stylesheet' />
+        <link rel='stylesheet' href='https://api.mapbox.com/mapbox-gl-js/plugins/mapbox-gl-geocoder/v2.2.0/mapbox-gl-geocoder.css' type='text/css' />
```

Comment:
One thing that doesn't matter--but helps us develop the right mindset--is to pay attention to details. 

Note in this diff we have mixed spacing after the colons in the CSS, mixed single- and double-quotes, some inline styles and some external styles. I'm sure this was a bit of a crazy mental process, but it helps to just develop the attention to detail that gets our brains in a better place to pay attention to detail in bigger places.